tools/website/web-pre.sh

```diff
@@ -46,6 +48,7 @@ echo "$(cat <<EOT
 ---
 title: Contributing
 type: docs
+original_path: CONTRIBUTING.md
```

Comment:
`original_path` > `originalpath` to quick fix it, or rename all other `originalpath` references to `original_path` to make it clearer.

Reply:
yeah my bad will do